packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -68,26 +65,16 @@ export class VaBreadcrumbs {
   }
 
   componentWillLoad() {
-    //  This renders slot content outside of a slot.
-    //  This is a workaround for the limitations of styling slotted nested elements.
-    //  See https://stackoverflow.com/questions/61626493/slotted-css-selector-for-nested-children-in-shadowdom-slot/61631668
     const anchorNodes = Array.from(this.el.querySelectorAll('a'));
-    this.breadcrumbs = anchorNodes.map(
-      (node: HTMLAnchorElement, index: number) => {
-        return (
-          <li>
-            <a
-              aria-current={
-                index === anchorNodes.length - 1 ? 'page' : undefined
-              }
-              href={node.attributes['href'].value}
-            >
-              {node.innerText}
-            </a>
-          </li>
-        );
-      },
-    );
+    anchorNodes.forEach((crumb: HTMLAnchorElement, index: number) => {
+      const li = document.createElement('li');
+      li.classList.add('va-breadcrumbs-li');
+      if (index === anchorNodes.length - 1) {
+        crumb.setAttribute('aria-current', 'page');
+      }
+      crumb.parentNode.replaceChild(li, crumb);
+      li.appendChild(crumb);
+    });
```

Comment:
This is cool, I'm glad this wrapping worked. Could you add a comment to the code that explains this a bit?

I also like the `.classList.add()` and think that we may be able to use that to help us manage the global styles in our `main.css`.

For example, we could do something like:

```js
      li.classList.add('vads-u-display--none medium-screen:vads-u-display--block');
```

I'm not saying that we should do that now, it's just food for thought. I think in order for something like that to work with all of the styling we're using here we'd need to spend some time on further developing our utility classes.